manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -82,9 +108,9 @@ public void stop(Container container) throws Exception {
     }
 
 
-    public void saveMangerConfig(Object managerConfiguration) {
+    public void saveManagerConfigFile(Object managerConfiguration) {
```

Comment:
I know it wasn't there before but this should throw an exception back to the reponse (500 server error)